gammapy/maps/axes.py

```diff
@@ -1203,6 +1239,30 @@ def __init__(self, axes, n_spatial_axes=None):
         self._axes = axes
         self._n_spatial_axes = n_spatial_axes
 
+    @property
+    def primary_axis(self):
```

Comment:
I'm not sure relying on the order works well, because in practice both cases will exist: lightcurves in energy bands and SED points in time bins. Right now the functionality is only used to derive a default for plotting. We could also just remove it and make `axis_name` a required argument for `.plot()`...

Reply:
This sounds good to me.
I guess the expectation from user will be that a LC is plotted as a LC and flux points as flux points. So one can plot without specifying the axis as long as the object is unidimensional. 
As soon as multiple non flat axes are present, the users should specify the primary one.

Reply:
Sounds good to me as well. I'll change the behaviour this way...